src/core/lv_obj.h

```diff
@@ -59,7 +59,7 @@ enum {
     LV_STATE_USER_4      =  0x8000,
 
     LV_STATE_ANY = 0xFFFF,    /**< Special value can be used in some functions to target all states*/
-};
+} lv_obj_state_t;
```

Comment:
```suggestion
} lv_state_t;
```
to keep it consistent with the enum values.